src/sagarShrikhande/arrayAndStrings/A23_P1_ArrayOps.java

```diff
@@ -0,0 +1,33 @@
+package sagarShrikhande.arrayAndStrings;
+
+public class A23_P1_ArrayOps {
+
+	int getSecondMaxInArray(int[] a, int n) {
+		if (a.length >= n) {
```

Comment:
your logic is very generic which is good, just FYI,
![image](https://user-images.githubusercontent.com/46770978/149667894-4db78c98-7df7-41a1-8267-1256f53d5953.png)